tests/models/test_multi_task_segmentor.py

```diff
@@ -12,7 +12,7 @@
 ON_GPU = toolbox_env.has_gpu()
 # The value is based on 2 TitanXP each with 12GB
 BATCH_SIZE = 1 if not ON_GPU else 16
-NUM_POSTPROC_WORKERS = 2 if not toolbox_env.running_on_travis() else 8
+NUM_POSTPROC_WORKERS = 2 if not toolbox_env.running_on_ci() else 8
```

Comment:
Can we try to increase `NUM_POSTPROC_WORKERS` as this is the limit on travis. May be GitHub actions provides more.

Reply:
Looks like a two core CPU virtual machine with 7GB RAM: https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners#supported-runners-and-hardware-resources

Reply:
Also, why not just use the `multiprocessing.cpu_count()` here instead of a hard-coded value of 2 or 8?

Reply:
Or better yet: `len(os.sched_getaffinity(0))`

Reply:
Fine with me

Reply:
I will address this on another PR.